cmd/dep/status.go

```diff
@@ -246,6 +378,21 @@ func runStatusAll(out outputter, p *dep.Project, sm *gps.SourceMgr) error {
 				PackageCount: len(proj.Packages()),
 			}
 
+			// Get children only for specific outputers
+			// in order to avoid slower status process
+			switch out.(type) {
+			case *dotOutput:
+				r := filepath.Join(p.AbsRoot, "vendor", string(proj.Ident().ProjectRoot))
+				ptr, err := gps.ListPackages(r, string(proj.Ident().ProjectRoot))
+
+				if err != nil {
+					return fmt.Errorf("analysis of %s package failed: %v", proj.Ident().ProjectRoot, err)
+				}
+
+				prm, _ := ptr.ToReachMap(true, false, false, nil)
+				bs.Children = prm.Flatten(false)
```

Comment:
@sdboyer we are doing some sort of "filtering".
https://github.com/golang/dep/pull/271/files#diff-0d223b461ecb49bf384a2c75a3893558R51
When we are generating Graphviz nodes we are only working with packages which are presented in the lock in this way for are excluding others.